### PR TITLE
Update link to management console in Applications page

### PR DESCRIPTION
**Pull request by demobox**
**Category:**

- [ ] Virtual machines
- [x] Kubernetes apps
- [ ] Container images

---

### CODE REVIEW

File: k8s/spinnaker/templates/spinnaker_application_manifest_middle_secured.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
     notes: |-
       # Manage your Spinnaker
-      [Open Cloud Shell](https://console.cloud.google.com/cloudshell/editor?shellonly=true&cloudshell_git_repo=https://github.com/GoogleCloudPlatform/click-to-deploy.git&cloudshell_working_dir=k8s/spinnaker/scripts/manage&cloudshell_tutorial=landing_page_expanded.md)
+      [Open Management Environment in Cloud Shell](https://console.cloud.google.com/cloudshell/editor?shellonly=true&cloudshell_git_repo=https://github.com/GoogleCloudPlatform/click-to-deploy.git&cloudshell_working_dir=k8s/spinnaker/scripts/manage&cloudshell_tutorial=landing_page_expanded.md)
```
Comment (demobox) [inline k8s/spinnaker/templates/spinnaker_application_manifest_middle_secured.yaml:3]: @dorbin Might be a bit long :-( Any suggestions?

Comment (demobox) [reply k8s/spinnaker/templates/spinnaker_application_manifest_middle_secured.yaml:3]: Hm, not terrible?

image

Comment (dorbin) [reply k8s/spinnaker/templates/spinnaker_application_manifest_middle_secured.yaml:3]: The current one I'm seeing live just says "Open Cloud Shell." 
How 'bout "Open Cloud Shell to manage Spinnaker"?

Comment (demobox) [reply k8s/spinnaker/templates/spinnaker_application_manifest_middle_secured.yaml:3]: That would also work, but I'd like to try to make it clear that Cloud Shell is an implementation detail, and what we're actually pointing users to is some kind of management interface. I.e. the link should be able to remain the same if we e.g. reimplemented this _without_ Cloud Shell in future.

**Review by duftler — APPROVED** [review]
LGTM.

**Pull request merged**